app/models/authentication.rb

```diff
@@ -2,7 +2,6 @@ class Authentication < ApplicationRecord
   include PasswordConcern
   include TenancyConcern
   include EventConcern
-  include AvailabilityStatusConcern
```

Comment:
Shouldn't we leave this in since we have the `availability_status` columns still? It's just the `status/status_details` ones that got removed.